pkg/ddl/job_table.go

```diff
@@ -138,10 +138,8 @@ func (d *ddl) getJob(se *sess.Session, tp jobType, filter func(*model.Job) (bool
 }
 
 func hasSysDB(job *model.Job) bool {
-	sNames := job2SchemaNames(job)
```

Comment:
I think we can remove `job2SchemaNames` after this update.

Reply:
Updated.